src/emulator/modules/SceTouch/SceTouch.cpp

```diff
@@ -29,36 +29,38 @@ static int peek_touch(HostState &host, SceUInt32 port, SceTouchData *pData) {
-        const uint32_t buttons = SDL_GetMouseState(&touch_pos_window.x, &touch_pos_window.y);
-        const uint32_t mask = (port == 1) ? SDL_BUTTON_RMASK : SDL_BUTTON_LMASK;
-        if ((buttons & mask) && host.gui.renderer_focused) {
-            SceIVector2 window_size = { 0, 0 };
-            SDL_Window *const window = SDL_GetMouseFocus();
-            SDL_GetWindowSize(window, &window_size.x, &window_size.y);
-
-            SceFVector2 scale = { 1, 1 };
-            if ((window_size.x > 0) && (window_size.y > 0)) {
-                scale.x = static_cast<float>(host.drawable_size.x) / window_size.x;
-                scale.y = static_cast<float>(host.drawable_size.y) / window_size.y;
-            }
-
-            const SceFVector2 touch_pos_drawable = {
-                touch_pos_window.x * scale.x,
-                touch_pos_window.y * scale.y
-            };
-
-            const SceFVector2 touch_pos_viewport = {
-                (touch_pos_drawable.x - host.viewport_pos.x) / host.viewport_size.x,
-                (touch_pos_drawable.y - host.viewport_pos.y) / host.viewport_size.y
-            };
-
-            if ((touch_pos_viewport.x >= 0) && (touch_pos_viewport.y >= 0) && (touch_pos_viewport.x < 1) && (touch_pos_viewport.y < 1)) {
-                pData->report[pData->reportNum].x = static_cast<uint16_t>(touch_pos_viewport.x * 1920);
-                pData->report[pData->reportNum].y = static_cast<uint16_t>(touch_pos_viewport.y * 1088);
-                ++pData->reportNum;
-            }
+    SceIVector2 touch_pos_window = { 0, 0 };
+    const uint32_t buttons = SDL_GetMouseState(&touch_pos_window.x, &touch_pos_window.y);
+    const uint32_t mask = (port == 1) ? SDL_BUTTON_RMASK : SDL_BUTTON_LMASK;
+    if ((buttons & mask) && host.gui.renderer_focused) {
+        SceIVector2 window_size = { 0, 0 };
+        SDL_Window *const window = SDL_GetMouseFocus();
+        SDL_GetWindowSize(window, &window_size.x, &window_size.y);
+
+        SceFVector2 scale = { 1, 1 };
+        if ((window_size.x > 0) && (window_size.y > 0)) {
+            scale.x = static_cast<float>(host.drawable_size.x) / window_size.x;
+            scale.y = static_cast<float>(host.drawable_size.y) / window_size.y;
+        }
+
+        const SceFVector2 touch_pos_drawable = {
+            touch_pos_window.x * scale.x,
+            touch_pos_window.y * scale.y
+        };
+
+        const SceFVector2 touch_pos_viewport = {
+            (touch_pos_drawable.x - host.viewport_pos.x) / host.viewport_size.x,
+            (touch_pos_drawable.y - host.viewport_pos.y) / host.viewport_size.y
+        };
+
+        if ((touch_pos_viewport.x >= 0) && (touch_pos_viewport.y >= 0) && (touch_pos_viewport.x < 1) && (touch_pos_viewport.y < 1)) {
+            pData->report[pData->reportNum].x = static_cast<uint16_t>(touch_pos_viewport.x * 1920);
+            pData->report[pData->reportNum].y = static_cast<uint16_t>(touch_pos_viewport.y * 1088);
```

Comment:
`1088` is not a typo right?

Reply:
i have think same after see that, no idea if error for want said 1080*

Reply:
Not a typo, Vita screen is 960x544. Touchscreen to handle it returns samples into the res x2 range (so 1920x1088).